README.md

```diff
@@ -38,49 +38,16 @@ We developed a patcher that attempts to fix all currently known affected mods (l
-- [JourneyMap](https://www.curseforge.com/minecraft/mc-mods/journeymap) (Issue introduced in 1.16.5-5.7.1 and fixed in 1.16.5-5.7.2 No other versions were effected)
-- [LanteaCraft / SGCraft](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1292427-lanteacraft)
-- [LogisticsPipes](https://www.curseforge.com/minecraft/mc-mods/logistics-pipes) (Only affects versions for Minecraft 1.4.7-1.7.10. Fixed in version 0.10.0.71 for MC 1.7.10, [relevant security advisory](https://github.com/RS485/LogisticsPipes/security/advisories/GHSA-mcp7-xf3v-25x3))
-- [Minecraft Comes Alive (MCA)](https://www.curseforge.com/minecraft/mc-mods/minecraft-comes-alive-mca) (Only affects versions for Minecraft 1.5.2-1.6.4)
-- [MattDahEpic Core (MDECore)](https://www.curseforge.com/minecraft/mc-mods/mattdahepic-core) (Only affects versions for Minecraft 1.8.8-1.12.2)
-- [mxTune](https://www.curseforge.com/minecraft/mc-mods/mxtune) (Only affects versions for Minecraft 1.12-1.16.5)
-- [p455w0rd's Things](https://www.curseforge.com/minecraft/mc-mods/p455w0rds-things)
-- [Project Blue](https://www.csse.canterbury.ac.nz/greg.ewing/minecraft/mods/ProjectBlue/)
-- [RadixCore](https://www.curseforge.com/minecraft/mc-mods/radixcore)
-- [RebornCore](https://www.curseforge.com/minecraft/mc-mods/reborncore) (affected versions: >= 3.13.8, <4.7.3, [relevant security advisory](https://github.com/TechReborn/RebornCore/security/advisories/GHSA-r7pg-4xrf-7mrm))
-- [SimpleAchievements](https://www.curseforge.com/minecraft/mc-mods/simple-achievements)
-- [SmartMoving](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1274224-smart-moving)
-- [Strange](https://www.curseforge.com/minecraft/mc-mods/strange)
-- [SuperMartijn642's Config Lib](https://www.curseforge.com/minecraft/mc-mods/supermartijn642s-config-lib) (Fixed in version 1.0.9, [relevant security advisory](https://github.com/SuperMartijn642/SuperMartijn642sConfigLib/security/advisories/GHSA-f4r5-w453-2jx6))
-- [Thaumic Tinkerer](https://www.curseforge.com/minecraft/mc-mods/thaumic-tinkerer) (Fixed in version 2.3-138 for Minecraft 1.7.2, versions for 1.6-1.6.4 remain affected, [relevant commit](https://github.com/Thaumic-Tinkerer/ThaumicTinkerer/commit))
-- [Tough Expansion](https://www.curseforge.com/minecraft/mc-mods/tough-expansion)
-- [ttCore](https://www.curseforge.com/minecraft/mc-mods/ttcore) (Only affects versions for Minecraft 1.7.10)
+Unlike the blog post stated above, there are plenty of mods that are and could be affected by this issue. Although some of them are already fixed in their latest versions, these mods were exploitable in at least one older verison. It is to be expected that several modpacks over the years just are not maintained anymore, but are still popular and loved within the community. Keeping this in-mind, we are trying to help those people who still love running those modpacks and strive to keeping them safe as they play. 
```

Comment:
I believe the last sentence of this paragraph should instead say "striving to keep" to maintain consistency with "trying to help" earlier in the sentence.

Reply:
Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20